### ISSUE TITLE
Using Multiple Training Data with TPOT

### ISSUE THREAD

**Issue by seanlaw** [EpistasisLab/tpot#828]
If I have multiple training sets (i.e., `X_train_1`, `Y_train_1`, `X_train_2`, `Y_train_2`), is it possible to use TPOT to fit the same model parameters to multiple training sets? I'm using Dask distributed for parallel model building.

In the documentation, for a single training set, I should do:

    from sklearn.externals import joblib
    import distributed.joblib
    from dask.distributed import Client

    # connect to the cluster
    client = Client('schedueler-address')

    # create the estimator normally
    estimator = TPOTClassifier(n_jobs=-1)

    # perform the fit in this context manager
    with joblib.parallel_backend("dask"):
        estimator.fit(X_train_1, Y_train_1)

However, it isn't clear how I could also fit `X_train_2` and `Y_train_2` in parallel with `X_train_1` and `Y_train_1` without having to wait for `X_train_1` to complete.

**Comment by weixuanfu** [comment]
It is possible. You could merge all the training set with the same number of features. Then using 'cv' parameter to set the customized train/test splits to evaluate the model on multiple training sets.

**Comment by seanlaw** [comment]
But wouldn't this just calculate a mean accuracy from the two datasets? I want each dataset to represent an independent estimator. So, I want to see how well each type of classifier performs on different datasets (they would have different features).

**Comment by rhiever** [comment]
If you're running TPOT with `n_jobs=-1`, isn't it using all of the system resources? So you'll want to wait until the first TPOT run finishes before starting the next.

Regardless, I think the existing best solution for this is to rewrite your above code as a function and call the function in separate terminals, e.g.,

```python
from sklearn.externals import joblib
import distributed.joblib
from dask.distributed import Client

def run_tpot(X_train, Y_train):
    # connect to the cluster
    client = Client('schedueler-address')

    # create the estimator normally
    estimator = TPOTClassifier(n_jobs=-1)

    # perform the fit in this context manager
    with joblib.parallel_backend("dask"):
        estimator.fit(X_train, Y_train)
    return estimator
```

then

terminal 1:

```python
# import/data setup
...
run_tpot(X_train_1, Y_train_1)
```

terminal 2:

```python
# import/data setup
...
run_tpot(X_train_2, Y_train_2)
```

etc.
